React/tic-tac/src/app/components/Login/index.js

```diff
@@ -0,0 +1,48 @@
+import React, { Fragment, Component } from 'react';
+import { Redirect } from 'react-router-dom';
+
+import style from './style.scss';
+
+class Login extends Component {
+  state = {
+    loggedIn: false
+  };
+
+  loginHandle() {
+    this.setState(() => ({ loggedIn: true }));
+  }
+
+  fragmentRender() {
+    return (
+      <Fragment>
+        <div className={style.overlay}>
+          <div className={`${style.login} ${style.fadeInDown} ${style.animated}`}>
```

Comment:
I'd rename the `login` class to something that gives you an idea of what's being displayed. Something like `loginModal` or `loginForm`.